assets/js/atomic/blocks/product/button/block.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * External dependencies
```

Comment:
Moved contents of Atomic Components to a block.js per atomic block. This is so we can reuse the same block for edit.js and frontend.js, and also allows frontend.js to see the defined blockAttributes so it can map dataset to block attributes correctly.